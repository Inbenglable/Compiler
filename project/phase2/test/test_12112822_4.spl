struct Foo{
    int a;
    float b;
    char c;
} makeFoo(int a, float b, char c){
    struct Foo foo;
    foo.a = a;
    foo.b = b;
    foo.c = c;
    return foo;
}

int main(){
    return makeFoo(114514,1919.81,'0');
}