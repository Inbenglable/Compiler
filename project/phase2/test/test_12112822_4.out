Error type 8 at line 14: Return value type mismatches the declared type
